### DISCUSSION TITLE
Windows 98 Mouse issue

### DISCUSSION THREAD

**Discussion by YD711YT** [utmapp/UTM#6842] [UTM Help (iOS)]
**Describe the issue**
The mouse pointer can't be used when booting windows 98 using floppy disks
**Configuration (required)**
* UTM Version: 4.5.4
* OS Version: iPadOS 18.1 Beta
* Device Model: iPad Air(5st Gen)
* Is it jailbroken (name jailbreak used)? No
* How did you install UTM? App Store

**Crash log**
N/A

**Debug log**
N/A(not a system but qemu problem)

**Upload VM**
Can't due to submiting on iPad

**Comment by osy** [comment]
Since you didn't upload your config.plist I am going to guess that you didn't choose "other -> legacy hardware" when creating a new vm
